lib/active_record/virtual_attributes/virtual_fields.rb

```diff
@@ -183,8 +183,11 @@ def instantiate(result_set, *_, &block)
           additional_attributes = result_set.first.keys
                                             .reject { |k| join_dep_keys.include?(k) }
                                             .reject { |k| join_root_aliases.include?(k) }
-                                            .map    { |k| [k, k] }
-          column_aliases += additional_attributes
+          column_aliases += if ActiveRecord.version.to_s < "6.0"
```

Comment:
@kbrock I'm good with this but looking at this PR I'm wondering if we should always use the same format for these version checks...  in other words flip this from `<` to `>=` and flip the if/else.  If you agree, maybe you can do this in a followup PR.  It's the only suggestion I can make.